Source/UI/NewUI/CommonUI/View/Core/TextFieldView.swift

```diff
@@ -36,6 +36,7 @@ class TextFieldView: UIView {
         textField.placeholder = style.placeHolder
         textField.textColor = style.textColor
         textField.tintColor = style.tintColor
+        textField.textAlignment = style.textAlignment
```

Comment:
Documentation shows UILabel has a different default from UITextField in relation to the alignment, so it may be better to only apply when provided in all cases. 

Having said this, could we pull out the textAlignment from PR and have a separate discussion on why / how it benefits?